### ISSUE TITLE
Declarations in extensions cannot override yet - Swift 4

### ISSUE THREAD

**Issue by lusabo** [groue/GRMustache.swift#47]
When I tried to migrate my app to use Xcode 9 + Swift 4 I received the error **Declarations in extensions cannot override yet** in the files: Box.swift, Formatter.swift and Foundation.swift.

Example:

In the file Box.swift in the code below occurs the error in the line: **open override var mustacheBox: MustacheBox {**.

```
/// GRMustache provides built-in support for rendering `NSNull`.
extension NSNull {
   
   open override var mustacheBox: MustacheBox {
        return MustacheBox(
            value: self,
            boolValue: false,
            render: { (info: RenderingInfo) in return Rendering("") })
    }
}
```
How to solve?

Thanks,
Luciano

**Comment by lusabo** [comment]
I put @objc before the **open** word.

**Comment by hamedmohammadi** [comment]
you can go in pod project select GRMustache and change compiling swift to swift3.4 and everything work .
this project need to support swift 4 but doesn't yet

**Comment by EMart86** [comment]
I've seen a PR with swift4 compatibility has been merged 28 days ago. Any chance that this is merged on master and pushed as pod soon?

**Comment by yurkins** [comment]
same issue :(

**Comment by fumito-ito** [comment]
GRMustache.swift 6.0.0 is out on Cocoapods. Some issues are fixed by #103, #104. Please note: `pod lib lint` is broken by xcodeproj 1.26.0. For more details, please see https://github.com/CocoaPods/CocoaPods/issues/12664.
